modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/BaseDistributionZoneManagerTest.java

```diff
@@ -61,7 +62,15 @@
  * Base class for {@link DistributionZoneManager} unit tests.
  */
 @ExtendWith(ConfigurationExtension.class)
-public class BaseDistributionZoneManagerTest extends BaseIgniteAbstractTest {
+public abstract class BaseDistributionZoneManagerTest extends BaseIgniteAbstractTest {
+    protected static final String ZONE_NAME = "zone1";
+
+    protected static final int ZONE_ID = 1;
+
+    protected static final long TIMEOUT_MILLIS = 10_000L;
```

Comment:
Please rename both TIMEOUT_MILLIS and TIMER_SECONDS it's not clear what it means.

Reply:
What names do you suggest?
TIMEOUT_MILLIS - use for IgniteTestUtils#waitForCondition
TIMER_SECONDS - use for scale up and down

Reply:
Did it